cypress/e2e/Auth/SignUp.cy.js

```diff
@@ -2,7 +2,7 @@ import { faker } from '@faker-js/faker';
 
 const presets = ['iphone-x', 'macbook-13'];
 
-describe('SignUp E2E Test', () => {
+describe('SignUp E2E Test', { tags: ['@auth', '@business:critical'] }, () => {
```

Comment:
I know the signup use-case is @contained, but I prefer to run it, just in case any of the tests that use the signup are skipped (because of the tags)